projects/plugins/jetpack/tests/e2e/bin/container-setup.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Exit if any command fails.
+set -e
+
+#####
+# This script is designed to be running inside the WP container
+# it is basically a hacky way to insert arbitrary PHP code into wp-config without messing with bash escaping etc.
+# Also, it creates a debug log file
+
+function usage {
+	echo "usage: $0 command"
+	echo "  wp-setup                     Setup wp-config"
+	echo "  gb-prerelease               Pull latest Guteberg pre-prelease"
+	echo "  -h | usage                   output this message"
+	exit 1
+}
+
+function wp_config {
+	touch wp-content/debug.log
+	chown www-data:www-data wp-content/debug.log
+
+	sed -i "/\/\* That's all, stop editing! Happy publishing. \*\//i\
+	define( 'E2E_REQUEST_URL', ( ! empty( \\\$_SERVER['HTTPS'] ) ? 'https://' : 'http://' ) . ( ! empty( \\\$_SERVER['HTTP_HOST'] ) ? \\\$_SERVER['HTTP_HOST'] : 'localhost' ) );\n\
+	define( 'WP_SITEURL', E2E_REQUEST_URL );\n\
+	define( 'WP_HOME', E2E_REQUEST_URL );\n\
+	" wp-config.php
+}
+
+function gb_prerelease {
+	ZIP_PATH=${2:-"wp-content/gutenberg.zip"}
```

Comment:
$2 here is the second argument passed to gb_prerelease, which is always empty because you don't pass any parameters on line 42.

If you change line 42 to
```bash
gb_prerelease "$2"
```
then you could make this
```suggestion
	ZIP_PATH=${1:-"wp-content/gutenberg.zip"}
```
to do what you intend here.

Reply:
Oh, I haven't realized it. Thanks. I fixed all the similar calls.